.coveragerc

```diff
@@ -631,6 +631,9 @@ omit =
     homeassistant/components/msteams/notify.py
     homeassistant/components/mullvad/__init__.py
     homeassistant/components/mullvad/binary_sensor.py
+    homeassistant/components/mutesync/__init__.py
+    homeassistant/components/mutesync/config_flow.py
```

Comment:
The config flow cannot be excluded from test coverage. Remove this line and ensure the config flow had a 100% test coverage.